lua/lvim/config/init.lua

```diff
@@ -67,6 +68,11 @@ local function handle_deprecated_settings()
     end
   end
 
+  -- lvim.lsp.override
+  if vim.tbl_contains(vim.tbl_keys(lvim.lsp), "override") then
```

Comment:
because we have `override = {}`  down there, this warning always shows up no matter what,
shouldn't we check if exists and the length is bigger than one or sth?

Reply:
Oops 😅